packages/ast/javascript/package.json

```diff
@@ -54,9 +54,10 @@
     "@shared-configs/prettier": "workspace:*",
     "@shared-configs/typescript": "workspace:*",
     "@sondr3/minitest": "^0.1.1",
+    "@types/node": "^16.11.7",
```

Comment:
I didn't realize, when a project that consumes `@codemod-cli/tests` doesn't install `@types/node`, they may get a TS error that says, `assert` has the type of `any`.